vars/download.yml

```diff
@@ -111,6 +113,12 @@ url:
     {%- endif -%}
   sha256_client_url:       "{{ static.clients_base_url }}/sha256sum.txt"
 
+  # Hardcoded for now
+  helm_client: "https://get.helm.sh/helm-v3.14.3-linux-amd64.tar.gz" # TODO: figure out how to get the correct arch
```

Comment:
One option would be to use [this script](https://web.archive.org/web/20240412204504/https://helm.sh/docs/intro/install/) to get the correct architecture helm binary. Up to you! @kazuhisya 

```
$ curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
$ chmod 700 get_helm.sh
$ ./get_helm.sh
```